FizzBuzz/Program.cs

```diff
@@ -10,75 +11,64 @@ static void Main(string[] args)
             Run();
         }
 
-        /* Using a string to store output instead of write command so only one write is needed and no extra boolean var */
         static void Run()
         {
             
             for (int i = 1; i < 300; i++)
             {
-                var output = "";
+                List<string> output = new List<string>();
                 if (i % 3 == 0)
                 {
-                    output += "Fizz";
+                    output.Add("Fizz");
 
                 }
 
                 if (i % 5 == 0)
                 {
-                    output += "Buzz";
+                    output.Add("Buzz");
                 }
 
                 if (i % 7 == 0)
                 {
-                    output += "Bang";
+                    output.Add("Bang");
                 }
 
                 if (i % 11 == 0)
                 {
-                    output = "Bong";
+                    output.Clear();
+                    output.Add("Bong");
                 }
 
                 if (i % 13 == 0)
                 {
-                    var firstB = output.IndexOf("B");
-                    if (firstB == -1)
+                    var firstB = output.FirstOrDefault(s => s.StartsWith("B"));
+                    if (firstB == null)
                     {
-                        output += "Fezz";
+                        output.Add("Fezz");
                     }
                     else
                     {
-                        output = output.Insert(firstB, "Fezz");
+                        output.Insert(output.IndexOf(firstB), "Fezz");
```

Comment:
Could you get the index of the first B word directly, instead of getting the word and then finding the index for it? 🙂